### PR TITLE
Add pr code coverage to ci

### LINKED ISSUE #4643: Add code coverage reporting and delta on PR
Jest provides coverage info, let's see where we want to surface that

- [ ] Add code coverage and/or delta to PR
- [ ] Do we want to gate anything in CI on this value?

### PR DESCRIPTION
**Pull request by drewbo**
## Changes proposed in this pull request:
- Add pr code coverage to ci 
- Close #4643

## security considerations
None

### CODE REVIEW

**Comment by cloud-gov-pages-operations** [comment]
## :robot: This is an automated code coverage report
  Total coverage (lines): 5.21%
  Coverage diff: 0%

**Comment by cloud-gov-pages-operations** [comment]
## :robot: This is an automated code coverage report
  Total coverage (lines): **5.48%**
  Coverage diff: **0.27%**

**Comment by cloud-gov-pages-operations** [comment]
## :robot: This is an automated code coverage report
  Total coverage (lines): **5.48%**
  Coverage diff: **0.27%**

**Comment by cloud-gov-pages-operations** [comment]
## :robot: This is an automated code coverage report
  Total coverage (lines): **5.48%**
  Coverage diff: **0.27%** :chart_with_upwards_trend:

File: ci/tasks/code-coverage-diff.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+diffcc=$(awk -v n1=$newcc -v n2=$oldcc -v OFMT="%.2f" 'BEGIN{print n1-n2}')
+emoji=$([ ${diffcc:0:1} == "-" ] && echo ":chart_with_downwards_trend:" || echo ":chart_with_upwards_trend:")
+
+body="## :robot: This is an automated code coverage report
+  Total coverage (lines): **$newcc%**
+  Coverage diff: **$diffcc%** $emoji
+"
+
+gh pr comment $branch --edit-last -b "$body" || gh pr comment $branch -b "$body"
```
Comment (apburnes) [inline ci/tasks/code-coverage-diff.sh:31]: Did we test this to see if it only edits the comment the bot generated? The `--edit-last` flag only edits the last comment created by the author?

**Review by apburnes — COMMENTED** [review]
Looks good to me. Just have a question to verify last edit.

File: ci/tasks/code-coverage-diff.sh (hunk 1 above)
Comment (drewbo) [reply ci/tasks/code-coverage-diff.sh:31]: I didn't do it extensively but [`--edit-last`](https://cli.github.com/manual/gh_pr_comment) is explicitly for the same author. If there isn't a previous comment, that command should fail and hit the next clause

**Review by apburnes — APPROVED** [review]

File: ci/tasks/code-coverage-diff.sh (hunk 1 above)
Comment (apburnes) [reply ci/tasks/code-coverage-diff.sh:31]: Good to go.

**Pull request merged**